extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -296,6 +296,7 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       this.carousel_.setAttribute('type', 'slides');
       this.carousel_.setAttribute('layout', 'fill');
       this.carousel_.setAttribute('loop', '');
+      this.carousel_.setAttribute('controls', 'controls');
```

Comment:
You don't actually need to set it to `'controls'`, you can just set it to the empty string like in the line above. 

```
this.carousel_.setAttribute('controls', '');
```